README.md

```diff
@@ -79,22 +80,27 @@ device available in those moments.
 ## Development
 
 ### Ruby Version
+
 This app uses Ruby version 2.6.3 and Rails version 5.2.3
 
 ### Setup
-* Clone the repo
-* run `bundle install`
-* run `rails db:create`
-* run `rake:db:migrate`
+
+- Clone the repo
```

Comment:
@abronkema can we use for [`bin/setup` ](https://github.com/rubyforgood/babywearing/blob/master/bin/setup) ?

Reply:
@dixpac - That seems pretty straightforward. Maybe you and I can talk through that in our call!

Reply:
Can we get this one in, with `bin/setup` as @dixpac said and then try to resolve this one as they are conflicting PR's: https://github.com/rubyforgood/babywearing/pull/85

Reply:
I'll merge this in as is, then I'll add the `bin/setup` in a separate PR.  Just so we can get this in and start resolving some stuff.